go/registry/api/api.go

```diff
@@ -54,6 +54,10 @@ var (
 	// runtime registation in the genesis document.
 	RegisterGenesisRuntimeSignatureContext = []byte("EkRunGen")
 
+	// RegisterUnfreezeNodeSignatureContext is the context used for
+	// unfreezing nodes.
+	RegisterUnfreezeNodeSignatureContext = []byte("EkUzNReg")
```

Comment:
I wonder if there is a way to ensure that we don't reuse contexts...  Alternatively I would be ok with changing the definition of a context to be between 1 to 255 (inclusive) octets.

Reply:
Yeah we should go for longer contexts, the current length is arbitrary. Also if possible it would be nice to enforce non-conflicting contexts, not sure how to do it -- associate context with a specific type and keep a map?

Reply:
#2311